src/components/EmptyState/EmptyState.tsx

```diff
@@ -1,41 +1,33 @@
-import { ReactComponent as EmptyIcon } from '@/assets/emptyIcon.svg'
-import styled, { css } from 'styled-components'
+import { EmptyIcon } from '@/components'
+import styled from 'styled-components'
 
 export type EmptyStateProps = {
-  message: string;
-  title: string;
+  message: string
+  title: string
 }
 
 const Wrapper = styled.div`
-  ${({ theme }) => css`
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    row-gap: ${theme.spacings.large};
-  `}
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: ${({ theme }) => theme.spacings.large};
 `
 
 const TextWrapper = styled.div`
-  ${({ theme }) => css`
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    row-gap: ${theme.spacings.xsmall};
-  `}
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  row-gap: ${({ theme }) => theme.spacings.xsmall};
 `
 
 const Title = styled.h1`
-  ${({ theme }) => css`
-    font-size: ${theme.fonts.sizes.xxlarge};
-  `}
+  font-size: ${({ theme }) => theme.fonts.sizes.xxlarge};
 `
 
 const Description = styled.p`
-  ${({ theme }) => css`
-    font-size: ${theme.fonts.sizes.xmedium};
-    color: ${theme.colors.grey[300]};
-    font-weight: 500;
-  `}
+  font-size: ${({ theme }) => theme.fonts.sizes.xmedium};
+  color: ${({ theme }) => theme.colors.grey[300]};
+  font-weight: 500;
 `
 
 export const EmptyState = ({ message, title }: EmptyStateProps) => {
```

Comment:
Melhorar composicao.

Reply:
Fixed in this [commit](https://github.com/joaooliveiradev/conduit-frontend/pull/43/commits/dca1f2b4737b9985172e564ce8757510e831e97d).